modules/nw-ingress-controller-nodeportservice-projects.adoc

```diff
@@ -0,0 +1,147 @@
++
+[source,terminal]
+----
+$ oc label node <node_name> ingresscontroller=<ingress_controller_custom_resource> <1>
+----
+<1> Where `<ingress_controller_custom_resource>` must match the value that you specified in the `nodePlacement` section of your `IngressController` CR.
+
+. Create the `IngressController` object:
++
+[source,terminal]
+----
+$ oc create -f <ingress_controller_cr>.yaml
+----
+
+. Find the port for the service that was created for the `IngressController` CR:
++
+[source,terminal]
+----
+$ oc get svc -n openshift-ingress
+----
++
+.Example output that shows port `80:32432/TCP` for the `router-nodeport-custom-ic3` service
+[source,terminal]
+----
+NAME                        TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                                     AGE
+router-internal-default     ClusterIP   172.30.195.74    <none>        80/TCP,443/TCP,1936/TCP                     223d
+router-nodeport-custom-ic3   NodePort    172.30.109.219   <none>        80:32432/TCP,443:31366/TCP,1936:30499/TCP   155m
+----
+
+. Use the `oc new-project` command to create a new project:
++
+[source,terminal]
+----
+$ oc new-project <ingress_controller_name>
+----
+
+. Enter the following command to label the additional Ingress Controller:
++
+[source,terminal]
+----
+$ oc label namespace <ingress_controller_name> ic=<ingress_controller_name> <1>
+----
+<1> Where `<ingress_controller_name>` must match the value that you specified in the `namespaceSelector` section of your Ingress Controller CR.
+
+. Create a new application in your cluster:
++
+[source,terminal]
+----
+$ oc new-app --image=quay.io/openshifttest/<service_name>:multiarch
+----
+
+. Create a `Route` object for a service, so that the pod can use the service to expose the application external to the cluster.
++
+[source,terminal]
+----
+$ oc expose svc/<service_name> --hostname=<custom_name>.<ingress_controller_name>.<domain_name> <1>
+----
++
+[NOTE]
+====
+You must specify the hostname and domain of your custom Ingress Controller in the `--hostname` argument. If you do not do this, the Ingress Operator uses the default Ingress Controller to serve all the routes for your cluster. 
+====
+
+. Update the `IngressController` CR to prevent the default Ingress Controller from managing the `NodePort`-type `Service`. The default Ingress Controller will continue to monitor all other cluster traffic.
++
+[source,terminal]
+----
+$ oc patch --type=merge -n openshift-ingress-operator ingresscontroller/default --patch '{"spec":{"namespaceSelector":{"matchExpressions":[{"key":"ic","operator":"NotIn","values":["<custom_name>]}]}}}'
+----
+
+.Verification
+
+. Verify that the wildcard DNS record is managed by the Ingress Controller. A successful connection means that your cluster can use IP addresses from external DNS servers for DNS resolution.
++
+[source,terminal]
+----
+$ curl <service_name>.<ingress_controller_name>.<domain_name>:<port> <1>
```

Comment:
```suggestion
$ curl <project_name>.<ingress_controller_name>.<domain_name>:<port> <1>
```

Reply:
so that's one way we could do it, but that's not what customers do.  A customer would do this:
`# oc get route
NAME              HOST/PORT                                             PATH   SERVICES          PORT       TERMINATION   WILDCARD
hello-openshift   hello-openshift.nodeportsvc.ipi-cluster.example.com          hello-openshift   8080-tcp                 None
oc curl hello-openshift.nodeportsvc.ipi-cluster.example.com:<port>
`

Reply:
note that "hello-openshift.nodeportsvc.ipi-cluster.example.com" is the HOST/PORT from the route results

Reply:
Removed:

````
$ curl <service_name>.<project_name>.<domain_name>:<port> <1>

<1> Where `<project_name>` must match the value that you specified in the `namespaceSelector` section of your custom Ingress Controller CR. For example, `openshift-service.custom-ic3.ipi-cluster.example.com:32432`.
````